tools/cli/go-whisk-cli/commands/activation.go

```diff
@@ -114,7 +114,11 @@ var activationGetCmd = &cobra.Command{
     PreRunE: setupClientConfig,
     RunE: func(cmd *cobra.Command, args []string) error {
         var field string
+        var err error
 
+        if args, err = lastFlag(args); err != nil {
+          return err
```

Comment:
The `lastFlag` function seems to have some decent debug that seems sufficient.  That said, when an error occurs, the final failure message to the user should be that this command failed with the failure reason.